docs/api/conformance/Open-Tests.md

```diff
@@ -0,0 +1,45 @@
+---
+id: Open-Tests
+sidebar_label: Open Tests
+title: Open Tests
+hide_title: true
+---
+
+# Open Tests
+<!-- markdownlint-disable MD033 -->
+
+## A Opens B  
+
+| App | Step            | Description                                              |
+|-----|-----------------|----------------------------------------------------------|
+| A   | 1. Opening App  | App A calls a function (see below) to open a second app, B |
+| A   | 2. Check Metadata | Ensure that the correct app was opened |
+
+- `AOpensB3`:  **A** uses an `AppMetadata` or `AppIdentifier` to open B, via:
+  - `fdc3.open({appId: “<app B ID>”})`
```

Comment:
@kriswest At the risk of being picky, there's a few places in this file where we seem to be using smart quotes rather than straight quotes - which is technically wrong for code samples, and usually something that happens by accident if partially authored in Word or SharePoint or something.

Reply:
Yes, those are in the current files and are not correct (this PR moved the files which is why its all showing up in the diff). Heres an example from main to compare with: https://github.com/finos/FDC3/blob/d5aab3f3cc1d98598cae9d6cdd063b36e509bcdf/toolbox/fdc3-conformance/Open-Tests.md?plain=1#L13-L14

They could do with correcting, yes.

Reply:
@kemerava also pointed out some consistency issues with the language that also needs fixing.

This PR has only addressed adding the existing files into the website and any preamble content needed - it didn;t go as far as correcting the content (yet?)

Reply:
Fixed all smart quotes